src/Text/index.js

```diff
@@ -67,7 +67,10 @@ export const StyledText = styled(({ element: TextElement, children, className, d
   a:not(${StyledTextLink}) {
     // TextLink in Text always win
     ${({ theme }) =>
-      getLinkStyle({ theme, type: TEXTLINK_TYPE_OPTIONS.PRIMARY })} // Get styles from TextLink
+      getLinkStyle({
+        theme,
+        type: TEXTLINK_TYPE_OPTIONS.PRIMARY,
+      })} // Get styles from TextLink;;;;;;;;;;
```

Comment:
What's this? 😅

Reply:
You should ask yourself, it's not my addition, I will look in to that if it's needed

Reply:
I think that it caused by running `yarn eslint --fix` because a lot of other files are linted even it should not 🤷‍♂️

Reply:
True, my apologies